cs/test/core/Util.cs

```diff
@@ -271,6 +273,22 @@ public static ArraySegment<byte> SerializeUnsafeCB<T>(T obj)
             return output.Data;
         }
 
+        public static ArraySegment<byte> SerializeUnsafeCBWithPtr<T>(T obj)
```

Comment:
But we also have:
```
Managed Serialize -> Managed Deserialize
Unmanaged Serialize -> Unmanaged Deserialize
```
Which I *think* should cover it. Although I will also extend the compatibility test, which covers more than current version roundtrip, to use all the streams.